src/constants/externalResources.ts

```diff
@@ -35,17 +53,7 @@ export const getLobsterTestLogUrl = (
         lineNum ? `#shareLine=${lineNum}` : ""
       }`;
 
-export const isLobsterLink = (url: string) => url.includes("/build/");
-export const getGithubPullRequestUrl = (
-  owner: string,
-  repo: string,
-  issue: number | string
-) => `https://github.com/${owner}/${repo}/pull/${issue}`;
-export const getLobsterTaskLink = (
-  logType: LogTypes,
-  taskId: string,
-  execution: number
-) =>
-  `${getLobsterURL()}/lobster/evergreen/task/${taskId}/${execution}/${logType}`;
+export const isLogkeeperLink = (link: string) =>
+  link.includes(`${deprecatedLogkeeperLobsterURL}/build`);
 
-export const deprecatedLogkeeperLobsterURL = "https://logkeeper.mongodb.org";
+export const getLogLink = (logPath: string) => `${getUiUrl()}${logPath}`;
```

Comment:
`rawDisplayURL` and `htmlDisplayURL` used to contain the baseURL. Do you know why it changed?

Reply:
I'm looking at the resolver for TestResult and I see that we include the baseURL for EVG testresults but not for cedar testresults. https://github.com/evergreen-ci/evergreen/blob/main/graphql/resolvers.go#L1306-L1313